### PR TITLE
add infra nodes and move components to infra nodes

### PR DESCRIPTION
**Pull request by qiliRedHat**
https://issues.redhat.com/browse/OCPQE-22074

### CODE REVIEW

**Comment by qiliRedHat** [comment]
@liqcui PTAL

File: reliability-v2/utils/openshift-qe-workers-infra-workload-commands.sh [hunk 1]
```diff
@@ -0,0 +1,771 @@
...
+#             CHANGE BELOW VARIABLE IF YOU WANT TO SET DIFFERENT VALUE               #
+######################################################################################
+SET_ENV_BY_PLATFORM=${SET_ENV_BY_PLATFORM:=$platform_type}
+echo SET_ENV_BY_PLATFORM is $SET_ENV_BY_PLATFORM
+case ${SET_ENV_BY_PLATFORM} in
+	aws)
+     #ARM64 Architecture:
+	   if [[ $node_arch == "arm64" ]];then
+	      if [[ ${scale_type} == "medium" ]];then
+                OPENSHIFT_INFRA_NODE_INSTANCE_TYPE=m6g.12xlarge
```
Comment (liqcui) [inline reliability-v2/utils/openshift-qe-workers-infra-workload-commands.sh:618]: the arm instance size still not correct, please download latest code from openshift/release.

Comment (liqcui) [reply reliability-v2/utils/openshift-qe-workers-infra-workload-commands.sh:618]: @qiliRedHat It seems the instance type of arm64 still not correct, The latest one https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-commands.sh#L606

Comment (qiliRedHat) [reply reliability-v2/utils/openshift-qe-workers-infra-workload-commands.sh]: @liqcui I updated it, thanks!

**Pull request merged**